pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mandible"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
I think it's OK not to bump the version number again since the previous version isn't being used anywhere yet. I made a few tags/releases for the versions that we have used for Nisar, and I'm thinking that in the future we should pin to a tag in our requirements.txt instead of a commit hash so it's a little more clear what's changing when we update. This way we can use the tags to bundle multiple PR's into a single version number bump rather than having to constantly bump the version in every PR.